modules/olm-accessing-images-private-registries.adoc

```diff
@@ -136,6 +138,7 @@ spec:
       interval: 30m
 ----
 <1> Add a `spec.secrets` section and specify any required secrets.
+<2> Specify the value of `legacy` or `restricted`. If the field is not set, the default value is `legacy`. In a future {product-title} release, it is planned that the default value will be `restricted`. If your catalog cannot run with `restricted` permissions, it is recommended that you manually set this field to `legacy`.
```

Comment:
FYI, for now, the four default catalog sources' SCC has been set to `restricted` as default.
```yaml
MacBook-Pro:~ jianzhang$ oc get catalogsource -n openshift-marketplace
NAME                  DISPLAY                TYPE   PUBLISHER      AGE
certified-operators   Certified Operators    grpc   Red Hat        3h58m
community-operators   Community Operators    grpc   Red Hat        3h58m
qe-app-registry       Production Operators   grpc   OpenShift QE   3h32m
redhat-marketplace    Red Hat Marketplace    grpc   Red Hat        3h58m
redhat-operators      Red Hat Operators      grpc   Red Hat        3h58m

MacBook-Pro:~ jianzhang$ oc -n openshift-marketplace get catalogsource -o yaml|grep securityContextConfig 
      securityContextConfig: restricted
      securityContextConfig: restricted
      securityContextConfig: restricted
      securityContextConfig: restricted
```

Reply:
@jianzhangbjz The default value of `securityContextConfig` is legacy. The catalogs that RH ships by default explicitly set `restricted` to override the default `legacy` value of `securityContextConfig`.

I don't think we need to talk about the RH shipped catalog configuration in this doc.